docs/protocol/src/concepts/validators.md

```diff
@@ -3,44 +3,52 @@
+                                 │             ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ┘        ┃      Tombstoned       ┃         │
+                                 │                       │          ┌──────▶┃     (Misbehavior)     ┃         │
+                                 │                       │          │       ┗━━━━━━━━━━━━━━━━━━━━━━━┛         │
+                                 │                       │          │                                         │
+┌ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─            │                       ▼          │                                         ▼
+      Validator      │      ┏━━━━━━━━━┓              ╔══════╗       │                                   ┏━━━━━━━━━━━┓
+│     Definition     ──────▶┃Inactive ┃─────────────▶║Active║───────┼────────────────────────────────┬─▶┃ Disabled  ┃
+   (in transaction)  │      ┗━━━━━━━━━┛              ╚══════╝       │                                │  ┗━━━━━━━━━━━┛
+└ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─            ▲                                  │       ┏━━━━━━━━━━━━━━━━━━━━━┓  │        │
+                                 │                                  └──────▶┃ Jailed (Inactivity) ┃──┘        │
+                                 │                                          ┗━━━━━━━━━━━━━━━━━━━━━┛           │
+                                 │                                                     │                      │
+                                 └─────────────────────────────────────────────────────┴──────────────────────┘
 ```
 
 Single lines represent unbonded stake, and double lines represent bonded stake.
 
-Validators become known to the chain either at genesis, or by means of a transaction with a `ValidatorDefinition` action in them. Validators transition through four states:
+Validators become known to the chain either at genesis, or by means of a transaction with a `ValidatorDefinition` action in them. Validators transition through five states:
 
-* **Inactive**, where the validator is not part of the consensus set, the stake in the validator's delegation pool is not bonded;
+* **Inactive**, where the validator is not part of the consensus set, and the stake in the validator's delegation pool is not bonded;
 * **Active**, where the validator is part of the consensus set, and the stake in the validator's delegation pool is bonded;
-* **Unbonding**, where the validator is not part of the consensus set, but the stake in the validator's delegation pool is still bonded;
-* **Slashed**, where the validator is not part of the consensus set, and the stake in the validator's delegation pool is not bonded.
+* **Jailed**, where the validator is not part of the consensus set, and the stake in the validator's delegation pool is not bonded;
+* **Tombstoned**, where the validator is not part of the consensus set, and the stake in the validator's delegation pool is not bonded;
+* **Disabled**, where the validator is not part of the consensus set, and the stake in the validator's delegation pool is not bonded.
```

Comment:
IMO the ", and the stake is/is not bonded" isn't useful here, now that we separate out bondingness and the state. I think this would be clearer if we just had each bullet point have a brief description of the state, like "Inactive: a validator whose delegation pool is too small to participate in consensus", and so on.